lab-christian/__test__/auth-router.test.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const request = require('superagent');
+const User = require('../model/user');
+const serverToggle = require('../lib/server-toggle');
+const server = require('../server');
+
+require('jest');
+
+const url = 'http://localhost:3000';
+
+const exampleUser = {
```

Comment:
consider using faker to generate your fake login info. This will help to ensure your tests check more possibilities.